hal/inc/cellular_hal_constants.h

```diff
@@ -41,6 +41,7 @@ typedef void (*_CELLULAR_SMS_CB_MDM)(void* data, int index);
     DEFINE_NET_PROVIDER( CELLULAR_NETPROV_TELEFONICA, "spark.telefonica.com", (23*60), (5684) ),  \
     DEFINE_NET_PROVIDER( CELLULAR_NETPROV_KORE_VODAFONE, "vfd1.korem2m.com", (23*60), (5684) ),  \
     DEFINE_NET_PROVIDER( CELLULAR_NETPROV_KORE_ATT, "10569.mcs", (23*60), (5684) ),  \
+    DEFINE_NET_PROVIDER( CELLULAR_NETPROV_TWILIO, "super", (5*60), (5684) ),  \		// XXX: check keepalive
```

Comment:
Also this AFAIK should already be up to 30 mins. Probably a good idea to set to 23 same as on other platforms.

Reply:
I tested as well and noticed that it's greater than `5 min`. Also there is a caveat that `30 min` only works with the server IPs that are supplied with.